kafka-client/src/main/java/dev/responsive/kafka/api/stores/ResponsiveKeyValueParams.java

```diff
@@ -46,7 +49,12 @@ public static ResponsiveKeyValueParams fact(final String name) {
   }
 
   public ResponsiveKeyValueParams withTimeToLive(final Duration timeToLive) {
-    this.timeToLive = timeToLive;
+    this.ttlProvider = TtlProvider.withDefaultTtl(timeToLive);
+    return this;
```

Comment:
Ah actually I was planning to remove this, I just didn't get around to cleaning it up everywhere yet. Sounds like this would be your preference as well?

Reply:
that was, but now I'm doubting myself. I think most situations that want TTL will be OK with just a default ttl 🤷 it's not like it adds that much surface area to the API since underneath the hood it uses the same mechanism.

Reply:
fine with me 🙂